.github/workflows/linux-checks.yaml

```diff
@@ -77,3 +77,87 @@ jobs:
         with:
           name: coverage-out
           path: ${{env.GOPATH}}/src/github.com/insolar/assured-ledger/ledger-core/v2/coverage.out
+  func-test-kuber:
+    name: func-test-kuber # don't change, this name is used in repository restricts!
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: /home/runner/work/
+      V2_PATH: /home/runner/work/assured-ledger/assured-ledger/ledger-core/v2
+      INSOLAR_IMAGE: ci:5000/assured-ledger
+      USE_MANIFESTS: ci
+      GOPROXY: https://proxy.golang.org,https://goproxy.io,direct
+      KUBECTL: kubectl --kubeconfig=/tmp/k3s.yaml
+      KUBECTL_VERSION: v1.18.2
+      KUBEVAL_VERSION: 0.15.0
+      K3S_TAG: v1.17.4-k3s1
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Set up go ${{env.GO_VERSION}}
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{env.GO_VERSION}}
```

Comment:
Is Go really required on the host machine that just uses k8s?

Reply:
Yep, until we move tests into pod
now tests are running in front of cluster